manager/integration/tests/test_debug.py

```diff
@@ -0,0 +1,97 @@
+import time
+import socket
+
+import longhorn
+
+from kubernetes import client as k8sclient, config as k8sconfig
+
+from common import get_apps_api_client # NOQA
+
+
+Mi = (1024 * 1024)
+Gi = (1024 * Mi)
+
+
+def test_uninstall_script():
+    apps_api = get_apps_api_client()
+    while True:
+        for i in range(10):
+            name = "ds-" + str(i)
+            try:
+                ds = apps_api.read_namespaced_daemon_set(name, "default")
+                if ds.status.number_ready == ds.status.number_ready:
+                    apps_api.delete_namespaced_daemon_set(name, "default")
+            except Exception:
+                apps_api.create_namespaced_daemon_set(
+                    "default", ds_manifest(name))
+
+
+def ds_manifest(name):
+    return {
+        'apiVersion': 'apps/v1',
+        'kind': 'DaemonSet',
+        'metadata': {
+            'name': name
+        },
+        'spec': {
+            'selector': {
+                'matchLabels': {
+                    'app': name
+                }
+            },
+            'template': {
+                'metadata': {
+                    'labels': {
+                        'app': name
+                    }
+                },
+                'spec': {
+                    'terminationGracePeriodSeconds': 10,
+                    'containers': [{
+                        'image': 'busybox',
+                        'imagePullPolicy': 'IfNotPresent',
+                        'name': 'sleep',
+                        'args': [
+                            '/bin/sh',
+                            '-c',
+                            'while true;do date;sleep 5; done'
+                        ],
+                    }]
+                }
+            },
+        }
+    }
+
+
+def test_debug_vol():
+    client = get_client()
+    for i in range(30):
+        vol_name = "vol" + str(i+1)
+        try:
+            client.create_volume(name=vol_name, size=str(10*Gi),
+                                 numberOfReplicas=3)
+        except Exception:
+            pass
+
+
+def get_client():
+    for i in range(30):
+        try:
+            k8sconfig.load_incluster_config()
+            lm_pods = k8sclient.CoreV1Api().list_pod_for_all_namespaces(
+                label_selector="app=longhorn-manager",
+                watch=False)
+            mgr_ips = []
+            for i in lm_pods.items:
+                mgr_ips.append(i.status.pod_ip)
+
+            # check if longhorn manager port is open before calling get_client
+            for ip in mgr_ips:
+                sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+                mgr_port_open = sock.connect_ex((ip, 9500))
+
+                if mgr_port_open == 0:
+                    url = 'http://' + ip + ':9500' + '/v1/schemas'
+                    return longhorn.from_env(url=url)
+        except Exception:
+            time.sleep(1)
```

Comment:
Can you please explain in brief the purpose of this file

Reply:
Sorry. This file shouldn't be in this PR. My bad!